tests/services/label.service.spec.ts

```diff
@@ -67,34 +68,37 @@ describe('LabelService', () => {
   });
 });
 
-describe('LabelService: parseLabelData()', () => {
+describe('LabelService: toLabel()', () => {
   beforeAll(() => {
     labelService = new LabelService(null);
-    labelList = labelService.parseLabelData(LabelConstant.SOME_TEAM_RESPONSE_PHASE_LABELS);
   });
 
   afterAll(() => {
     labelService = null;
   });
 
-  it('should be response.Accepted label', () => {
-    expect(labelList[0].labelCategory).toBe(LabelConstant.RESPONSE);
-    expect(labelList[0].labelValue).toBe(LabelConstant.RESPONSE_ACCEPTED);
-    expect(labelList[0].labelColor).toBe(LabelConstant.COLOR_RESPONSE_ACCEPTED);
+  it('should be severity.Low label', () => {
```

Comment:
Just a nitpick, the string parameter in 'it' should be describing what 'LabelService: toLabel()' function should be doing. Something like ' should convert a severity.Low GithubLabel object to Label object' would be more suitable here. 
You can also merge the below tests into just 1 'it' block as well, using something like 'should convert a GithubLabel object to a corresponding Label object', since they are all testing the same functionality of the function.

Reply:
You can refer to this for more information: https://jasmine.github.io/tutorials/your_first_suite